LamestWebserver/LamestWebserver/Core/Logger.cs

```diff
@@ -339,5 +443,32 @@ public enum ELoggingLevel : byte
             /// </summary>
             Trace = 0
         }
+
+        /// <summary>
+        /// Flags for output Sources
+        /// </summary>
+        [Flags]
+        public enum EOutputSource : byte
+        {
+            /// <summary>
+            /// No Logging output at all
+            /// </summary>
+            None = 0,
+
+            /// <summary>
+            /// Write To Console
+            /// </summary>
+            Console = 1,
+
+            /// <summary>
+            /// Write into File
+            /// </summary>
+            File = 2,
+
+            /// <summary>
+            /// Write into Costume definde Streams
```

Comment:
![image](https://user-images.githubusercontent.com/535578/31386876-ba66e38a-ae0b-11e7-95b7-9a2ee2e34caa.png)

Reply:
i need to install my spell checker again 😆